### PR TITLE
comply PEP-8 pycodestyle

### PR DESCRIPTION
**Pull request by CaitinZhao**
Thank you for your contribution to the MindYOLO repo.
Before submitting this PR, please make sure:

- [x] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindyolo/blob/master/CONTRIBUTING.md)
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [x] You have added unit tests

## Motivation

comply PEP-8 pycodestyle for MindYOLO

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: demo/predict.py [hunk 1]
```diff
@@ -1,83 +1,138 @@
...
-import cv2
-import numpy as np
 from datetime import datetime
 
+import cv2
 import mindspore as ms
-from mindspore import nn, context, Tensor
+import numpy as np
+import yaml
+from mindspore import Tensor, context, nn
```
Comment (zhanghuiyao) [inline demo/predict.py:12]: 对mindspore的import是否连在一起好一些

File: mindyolo/models/heads/yolov7_head.py [hunk 2]
```diff
@@ -119,69 +145,96 @@ def __init__(self, nc=80, anchors=(), stride=(), ch=()):  # detection layer
...
+        self.anchor_grid = Parameter(
+            Tensor(anchor_grid, ms.float32), requires_grad=False
+        )  # shape(nl,1,na,1,1,2)
+        self.convert_matrix = Parameter(
+            Tensor(
+                [[1, 0, 1, 0], [0, 1, 0, 1], [-0.5, 0, 0.5, 0], [0, -0.5, 0, 0.5]],
+                dtype=ms.float32,
+            ),
+            requires_grad=False,
+        )
```
Comment (zhanghuiyao) [inline mindyolo/models/heads/yolov7_head.py:168]: 是否保留原来的方式更直观、易读一些

File: mindyolo/models/heads/yolox_head.py [hunk 3]
```diff
@@ -1,17 +1,26 @@
 import math
 
 import mindspore as ms
-from mindspore import nn, ops, Tensor
+from mindspore import Tensor, nn
 from mindspore import numpy as mnp
+from mindspore import ops
```
Comment (zhanghuiyao) [inline mindyolo/models/heads/yolox_head.py:6]: 是否应该合并为一个，保持原代码

File: mindyolo/models/losses/yolov3_loss.py [hunk 4]
```diff
@@ -1,37 +1,46 @@
...
         super(YOLOv3Loss, self).__init__()
         self.hyp_box = box
         self.hyp_obj = obj
         self.hyp_cls = cls
         self.hyp_anchor_t = anchor_t
-        self.nc = nc                    # number of classes
+        self.nc = nc  # number of classes
         self.na = len(anchors[0]) // 2  # number of anchors
-        self.nl = len(anchors)          # number of layers
+        self.nl = len(anchors)  # number of layers
```
Comment (zhanghuiyao) [inline mindyolo/models/losses/yolov3_loss.py:44]: 是否原来的形式更美观一些

File: demo/predict.py (hunk 1 above)
Comment (CaitinZhao) [reply demo/predict.py]: done

File: mindyolo/models/heads/yolov7_head.py (hunk 2 above)
Comment (CaitinZhao) [reply mindyolo/models/heads/yolov7_head.py]: 行长度改成120了

File: mindyolo/models/heads/yolox_head.py (hunk 3 above)
Comment (CaitinZhao) [reply mindyolo/models/heads/yolox_head.py]: done

File: mindyolo/models/losses/yolov3_loss.py (hunk 4 above)
Comment (CaitinZhao) [reply mindyolo/models/losses/yolov3_loss.py:30]: 统一刷成只留两个空格了，也不影响美观性

**Review by zhanghuiyao — APPROVED** [review]

**Review by SamitHuang — APPROVED** [review]
looks better

**Pull request merged**